calico_cni/policy_drivers.py

```diff
@@ -338,28 +356,40 @@ def get_policy_driver(k8s_pod_name, k8s_namespace, net_config):
     policy_config = net_config.get(POLICY_KEY, {})
     network_name = net_config["name"]
     policy_type = policy_config.get("type")
+    supported_policy_types = [None,
+                              POLICY_MODE_KUBERNETES,
+                              POLICY_MODE_KUBERNETES_ANNOTATIONS]
+    if policy_type not in supported_policy_types:
+        print_cni_error(ERR_CODE_GENERIC,
```

Comment:
So, this is going to print out "supported values: [None, k8s-annotations, ...]" and the value `None` is not actually supported.  I think it shouldn't include `None` when printing out the list.